pkg/fleet/installer/env/env.go

```diff
@@ -293,6 +298,22 @@ func parseApmLibrariesEnv() map[ApmLibLanguage]ApmLibVersion {
 	return apmLibrariesVersion
 }
 
+// Checks if the machine the installer is currently on is running centos 6
+func DetectCentos6() bool {
+	sources := []string{
+		"/etc/system-release",
+		"/etc/centos-release",
+		"/etc/redhat-release",
+	}
+	for _, s := range sources {
+		b, _ := os.ReadFile(s)
+		if bytes.Contains(b, []byte("CentOS release 6")) {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Can't we use libc version? Maybe we could support other OSes in that case OOTB too.